relayer-cli/src/commands/keys/clean.rs

```diff
@@ -0,0 +1,73 @@
+use abscissa_core::{Command, Options, Runnable};
+use anomaly::BoxError;
+
+use ibc::ics24_host::identifier::ChainId;
+use ibc_relayer::{
+    config::{ChainConfig, Config},
+    keyring::{KeyRing, Store},
+};
+
+use crate::application::app_config;
+use crate::conclude::Output;
+
+#[derive(Clone, Command, Debug, Options)]
+pub struct KeysCleanCmd {
+    #[options(free, required, help = "identifier of the chain")]
+    chain_id: ChainId,
+
+    #[options(
+        short = "n",
+        help = "name of the key (defaults to the `key_name` defined in the config)"
+    )]
+    name: Option<String>,
```

Comment:
Should we also add an optional `--all` flag that deletes all keys for a chain?

Reply:
Good idea.

If I were a user, I would rather not have any default here. I would prefer to _always_ specify which key to delete (or all keys). So I would suggest we can even remove the part "(defaults to the `key_name` defined in the config)". Then this CLI would expect one out of two options


- `--name/-n KEY-NAME` or
- `--all/-a`

Reply:
Done... 👍